### PR TITLE
remove trailing commas for #35925

### PR DESCRIPTION
**Pull request by Anshoo-Rajput**
- [x] pandas/io/parquet.py
- [x] pandas/io/parsers.py
- [x] pandas/io/pytables.py

### CODE REVIEW

**Comment by rajanshoo25** [comment]
xref #35925

**Comment by simonjayhawkins** [comment]
Thanks @rajanshoo25 for the PR. generally lgtm.

File: pandas/io/parsers.py [hunk 1]
```diff
@@ -1967,9 +1967,7 @@ def _do_date_conversions(self, names, data):
 
 
 class CParserWrapper(ParserBase):
-    """
-
-    """
+    """"""
```
[the remark this answers is not in the document]
Comment (rajanshoo25) [reply pandas/io/parsers.py]: done! I removed that blank doctring.

**Comment by simonjayhawkins** [comment]
@rajanshoo25 CI / Checks (pull_request) failing. need to run black again after the changes

```
black, version 19.10b0
Checking black formatting
would reformat /home/runner/work/pandas/pandas/pandas/io/parsers.py
Oh no! 💥 💔 💥
1 file would be reformatted, 1122 files would be left unchanged.
```

**Review by MarcoGorelli — APPROVED** [review]
looks good to me

**Comment by simonjayhawkins** [comment]
Thanks @rajanshoo25

**Comment by rajanshoo25** [comment]
> Thanks @rajanshoo25

Thanks a lot! @simonjayhawkins for guiding me. Actually I am new in open-source contribution.

**Pull request merged**